worker-versioning/src/example.ts

```diff
@@ -0,0 +1,120 @@
+  const firstWorkflowID = 'worker-versioning-first_' + uuid4();
+  const firstWorkflow = await client.workflow.start(versioningExample, {
+    workflowId: firstWorkflowID,
+    taskQueue,
+    workflowExecutionTimeout: '5 minutes',
+  });
+
+  // Signal the workflow to drive it
+  await firstWorkflow.signal(proceeder, 'go');
+
+  // Give a chance for the worker to process the signal
+  await new Promise((resolve) => setTimeout(resolve, 3000));
+
+  // Add a new compatible version to the queue
+  await client.taskQueue.updateBuildIdCompatibility(taskQueue, {
+    operation: 'addNewCompatibleVersion',
+    buildId: '1.1',
+    existingCompatibleBuildId: '1.0',
+  });
+
+  // Stop the old worker, and start a 1.1 worker. We do this to speed along the example, since the
+  // 1.0 worker may continue to process tasks briefly after the version update.
+  worker1.shutdown();
+  await worker1Run;
+  const worker11 = await Worker.create({
+    workflowsPath: require.resolve('./workflowsV11'),
+    activities,
+    taskQueue,
+    buildId: '1.1',
+    useVersioning: true,
+  });
+  const worker11Run = worker11.run();
+
+  // Continue driving the workflow. Take note that the new version of the workflow run by the 1.1
+  // worker is the one that takes over! You might see a workflow task timeout, if the 1.0 worker is
+  // processing a task as the version update happens. That's normal.
+  await firstWorkflow.signal(proceeder, 'go');
+
+  // Add a new *incompatible* version to the task queue, which will become the new overall default
+  // for the queue.
+  await client.taskQueue.updateBuildIdCompatibility(taskQueue, {
+    operation: 'addNewIdInNewDefaultSet',
+    buildId: '2.0',
+  });
+  // Start a 2.0 worker
+  const worker2 = await Worker.create({
+    workflowsPath: require.resolve('./workflowsV2'),
+    activities,
+    taskQueue,
+    buildId: '2.0',
+    useVersioning: true,
+  });
+  const worker2Run = worker2.run();
+
+  // Start a new workflow, note that it will run on the new 2.0 version, without the client
+  // invocation changing at all!
+  const secondWorkflowID = 'worker-versioning-second_' + uuid4();
+  const secondWorkflow = await client.workflow.start(versioningExample, {
+    workflowId: secondWorkflowID,
+    taskQueue,
+    workflowExecutionTimeout: '5 minutes',
+  });
+
+  // Drive both workflows once more before concluding them
+  await firstWorkflow.signal(proceeder, 'go');
+  await secondWorkflow.signal(proceeder, 'go');
+  await firstWorkflow.signal(proceeder, 'finish');
+  await secondWorkflow.signal(proceeder, 'finish');
+
+  // Lastly we'll demonstrate how you can use the gRPC api to determine if certain build IDs are
+  // ready to be retied. There's more information in the documentation, but here's a quick example
+  // that will show us that we can retire the 1.0 worker:
+  const reachability = await client.taskQueue.getReachability({
+    buildIds: ['1.0'],
+  });
+  console.log('Reachability:', reachability);
```

Comment:
```suggestion
  console.log('Reachability:', reachability.buildIdReachability['1.0'].taskQueueReachability);
```

as is, logs:

Reachability: { buildIdReachability: { '1.0': { taskQueueReachability: [Object] } } }